web/src/bus.rs

```diff
@@ -8,11 +8,18 @@ use std::rc::Rc;
 ///
 /// [`Application`]: trait.Application.html
 #[allow(missing_debug_implementations)]
-#[derive(Clone)]
 pub struct Bus<Message> {
     publish: Rc<Box<dyn Fn(Message, &mut dyn dodrio::RootRender)>>,
 }
 
+impl<Message> Clone for Bus<Message> {
+    fn clone(&self) -> Self {
+        Self {
+            publish: Rc::clone(&self.publish),
```

Comment:
It looks like you discovered the same thing I did, where #[derive(Clone)] seemsto generate a wonky implementation of clone here, which only kicks in if Message is Clone -- even though all the fields are trivially cloneable unconditionally. And the solution isto just manually implement an unconditional clone.